converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java

```diff
@@ -72,7 +72,7 @@ protected DecodeResult decode(Element element, Node thisNode) {
 				setNationalProviderIdOnNode(element, thisNode);
 			}
 			if (ENTITY_VIRTUAL_GROUP.equals(entityType)) {
-				setVirtualGroupOnNode(element, thisNode);
+				setApmEntityIdOnNode(element, thisNode, ClinicalDocumentDecoder.ENTITY_ID);
```

Comment:
Naming is weird on these. We have a VG entity, but are setting an APM entity id on that record?

Reply:
Yeah, as I understood it, it was the entity id name. I look into setting this to be more generic it's not really an entity id but is a practice id I believe.

Reply:
@KyleApfel after a bit more research, what's happening is there is an identifier in QRDA-III called a Virtual Group Identifier. it looks something like the following when converted into QPP-JSON: 
`"performanceYear": 2020, "entityType": "virtualGroup", "entityId": "x12345",`
I think when Virtual Group was first introduced to the converter, it was thought that the entityId from APMS and the practice ID from CPC+ submissions are all the same. Their is more underlying code that needs to be re-worked and I think we should address it in a separate ticket as the scope of this task was just to properly encode entity Id's for MIPS APMS.

Reply:
I will add a name change from `setApmEntityIdOnNode` to `setEntityIdOnNode` as both virtual group and apm's need to have that entity id field. And then add a separate clean up ticket to separate how it finds the virtual group identifier as opposed to the  apm identifier.